### PR TITLE
Added intended use in the admin's introduction.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -5,11 +5,15 @@ The Django admin site
...
-reads metadata in your model to provide a powerful and production-ready
-interface that content producers can immediately use to start adding content to
-the site. In this document, we discuss how to activate, use and customize
-Django's admin interface.
+One of the most powerful parts of Django is the automatic admin interface.
+
+The admin reads metadata in your model to provide a quick and dirty interface
+that trusted users can use to start adding content to the site. It's often used
+as an organization's internal tool to manage a site. Don't try to build your
+entire site in the admin!
```
Comment (sasha0) [inline docs/ref/contrib/admin/index.txt]: Maybe "It's **highly** recommended to use Django admin site only as an administrative tool for managing content. It's not intended for building frontend." or something like that? Just tried to tone down last sentence a little bit.

File: docs/ref/contrib/admin/index.txt [hunk 2]
```diff
@@ -5,11 +5,15 @@ The Django admin site
...
    :synopsis: Django's admin site.
 
-One of the most powerful parts of Django is the automatic admin interface. It
-reads metadata in your model to provide a powerful and production-ready
-interface that content producers can immediately use to start adding content to
-the site. In this document, we discuss how to activate, use and customize
-Django's admin interface.
+One of the most powerful parts of Django is the automatic admin interface.
+
+The admin reads metadata in your model to provide a quick and dirty interface
```
Comment (MarkusH) [inline docs/ref/contrib/admin/index.txt]: "... read the metadata from your models to provide ..."

File: docs/ref/contrib/admin/index.txt (hunk 1 above)
Comment (MarkusH) [inline docs/ref/contrib/admin/index.txt]: :+1:

**Comment by mdewhirst** [comment]
--- C:/Users/mike/doc/django/admin/old.txt	Tue Feb  9 10:19:34 2016
+++ C:/Users/mike/doc/django/admin/new.txt	Tue Feb  9 10:19:15 2016
@@ -1 +1,3 @@
-One of the most powerful parts of Django is the automatic admin interface. It reads metadata in your model to provide a powerful and production-ready interface that content producers can immediately use to start adding content to the site. In this document, we discuss how to activate, use and customize Django’s admin interface.

+One of the most powerful parts of Django is the automatic admin interface. It reads metadata in your models to provide an interface that trusted content producers can immediately use to start adding content to the site. 
+
+In this document, we discuss how to activate, use and customize Django’s admin interface.

**Comment by olivierdalang** [comment]
Downgrading Django's admin from "powerful and production ready" to "quick and dirty" is very harsh... And I fear this will not only lower user's expectations, but also dev's carefulness !

Django's admin is and should aim to remain a great and solid admin. It's just not meant to be used as a front-end.

**Comment by mdewhirst** [comment]
The Admin is brilliant. It is certainly not "quick and dirty" and that is why I suggested the above changes. But I now realise that my suggested change uses the oxymoron "trusted users". Someone will need to come up with something better.

It may not have been designed as a front end but it works very well as one. It provides a secure and user-segregated view of the data. The *only* shortcoming I can see is an inability to restrict permission assignation to superusers. Given that, I would have *no* hesitation saying "powerful and production ready"

In fact there is nothing wrong with "powerful" and if I was making the above suggestion again I would include "powerful".

File: docs/ref/contrib/admin/index.txt [hunk 3]
```diff
@@ -5,11 +5,15 @@ The Django admin site
...
-One of the most powerful parts of Django is the automatic admin interface. It
-reads metadata in your model to provide a powerful and production-ready
-interface that content producers can immediately use to start adding content to
-the site. In this document, we discuss how to activate, use and customize
-Django's admin interface.
+One of the most powerful parts of Django is the automatic admin interface.
+
+The admin reads metadata from your models to provide a quick and dirty
+interface that trusted users can use to manage content on your site. It's
+recommended use is as an organization's internal management tool. It's not
```
Comment (alasdairnicol) [inline docs/ref/contrib/admin/index.txt]: Its recommended use is (no apostrophe)

**Comment by timgraham** [comment]
I made some revisions based on the feedback.

Mike, there's an open ticket if you're interested in contributing to the superuser issue you mentioned: https://code.djangoproject.com/ticket/23559

**Comment by mdewhirst** [comment]
Thanks Tim.

I worry that the description of the Admin sells it short. Maybe Django 
has reached critical mass and we no longer have to worry about 
non-Django user perceptions. The Admin is actually brilliant kit and 
works as advertised. It has always been a key advantage for Django.

I can see also from a developer's perspective that we need to tell it 
how it is.

It might be a mistake to over-correct.

My 2c

Mike

**Comment by timgraham** [comment]
I [wrote to django-developers](https://groups.google.com/d/topic/django-developers/479TIjEk1_g/discussion) to continue the discussion.

**Comment by mdewhirst** [comment]
Tim

I didn't realise you are Django Admin.

Let me congratulate you on a fabulous job of work. It is no mean feat.

For what its worth I agree with Russell in his reply on the devs list. 
You should be resting on your laurels not "getting weary". I've been 
there and it is no place to be. You're probably one of the very few 
people who can fix it when it breaks so you should not be "adding hook 
after hook for customizing every little thing." End of story.

If it wasn't for the Admin I would not have got into Django. So thank 
you very much. Have a look at www.sharedsds.com (a very rudimentary 
mezzanine site) to see what I did with the Admin.

I recognise I have to leave it behind sooner or later but when I do 
that'll be four years of work. And I worry about the "second system 
syndrome" claiming me.

Thanks again.

Mike

**Pull request merged**